.github/workflows/build.ps1

```diff
@@ -0,0 +1,79 @@
+#*****************************************************************************#
+#    Copyright (C) 2025 Julian Xhokaxhiu                                      #
+#                                                                             #
+#    This file is part of Field-Map-Editor                                    #
+#                                                                             #
+#    Field-Map-Editor is free software: you can redistribute it and\or modify #
+#    it under the terms of the GNU General Public License as published by     #
+#    the Free Software Foundation, either version 3 of the License            #
+#                                                                             #
+#    Field-Map-Editor is distributed in the hope that it will be useful,      #
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of           #
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the            #
+#    GNU General Public License for more details.                             #
+#*****************************************************************************#

```

Comment:
Also feel free to change this entire block to whatever license you apply for FME